extensions/common/iam/identity-trust/identity-trust-issuers-configuration/src/main/java/org/eclipse/edc/identitytrust/issuer/configuration/TrustedIssuerConfigurationExtension.java

```diff
@@ -29,12 +30,20 @@
 
 import static org.eclipse.edc.identitytrust.issuer.configuration.TrustedIssuerConfigurationExtension.NAME;
 
+/**
+ * This IATP extension makes it possible configure a list of trusted issuers, that will be matched against the Verifiable Credential issuers.
```

Comment:
This IATP extension makes it possible TO configure a list of trusted issuers ...